pkg/providers/vsphere/virtualmachine/backup.go

```diff
@@ -58,12 +58,8 @@ func BackupVirtualMachine(opts BackupVirtualMachineOptions) (result error) {
 	}()
 
 	resVM := res.NewVMFromObject(opts.VcVM)
-	moVM, err := resVM.GetProperties(opts.VMCtx, []string{"config.extraConfig"})
-	if err != nil {
-		opts.VMCtx.Logger.Error(err, "failed to get VM properties for backup")
-		return err
-	}
-	curExCfg := pkgutil.OptionValues(moVM.Config.ExtraConfig)
+
+	curExCfg := pkgutil.OptionValues(opts.VMCtx.MoVM.Config.ExtraConfig)
```

Comment:
TBH I'm not a huge fan that the VM properties are being carried along in the context, and there are some places where it is cleared. Can we add back the fetching if the field aren't set?